tests/tools/oesign/test-enclave/enclave/enc.c

```diff
@@ -13,11 +13,10 @@
 #include "oesign_test_t.h"
 
 /* Null-terminated hex string buffer size with 2 char per byte */
-const size_t OE_KSS_ID_HEX_BUFFER_SIZE = sizeof(oe_uuid_t) * 2 + 1;
+#define OE_KSS_ID_HEX_BUFFER_SIZE (sizeof(oe_uuid_t) * 2 + 1)
```

Comment:
I'm curious about what warning in clang-14 required changing from `const size_t` to a macro?

Reply:
```
error: variable length array folded to constant array as an extension [-Werror,-Wgnu-folding-constant]
    char unformatted_string[OE_KSS_ID_HEX_BUFFER_SIZE] = {0};
         ^
```